index.html

```diff
@@ -0,0 +1,257 @@
+            <li>Ruby</li>
+            <li>Ruby on Rails</li>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>ERB</li>
+            <li>RSPec</li>
+          </ul>
+          <a href="https://github.com/Spayco/katchy/" class="card-link link" target="_blank"
+            rel="noreferrer noopener">See project</a>
+        </div>
+      </div>
+
+    </div>
+    <div class="about-me">
+      <section class="about-me-section">
+        <h2>About me</h2>
+        <p class="about-me-text">
+          I am a skilled Full-Stack Web Developer who loves making pages with
+          beautiful UI. I have studied at Microverse, and I am experienced in
+          remote work.
+          <br />
+          I've also been a good helping hand to a lot of developers in
+          different communities!
+          <br />
+          if you'd like to get in contact about potential job opportunities,
+          please reach out!
+        </p>
+        <a href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+          class="link" target="_blank" rel="noreferrer noopener">
+          Get my resume
+        </a>
+        <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+      </section>
+      <hr />
+      <div class="about-me-section stacks">
+        <ul>
+          <li class="languages list-title">
+            Languages
+            <ul>
+              <li>JavaScript</li>
+              <li>TypeScript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Sass</li>
+            </ul>
+          </li>
+        </ul>
+        <ul>
+          <li class="frameworks list-title">
+            Frameworks
+            <ul>
+              <li>React & Redux</li>
+              <li>Material-UI</li>
+              <li>Jest</li>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+            </ul>
+          </li>
+        </ul>
+        <ul>
+          <li class="skills list-title">
+            Skills
+            <ul>
+              <li>GitHub</li>
+              <li>CodePen</li>
+              <li>GitLab</li>
+              <li>Terminal</li>
+              <li>WSL</li>
+              <li>Remote Pair-Programming</li>
+              <li>Teamwork</li>
+              <li>Mentoring</li>
+              <li>Code Reviews</li>
+              <li>Mobile/Responsive Development</li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+      <section class="contact">
+        <div class="contact-container">
+          <p class="contact-P">I'm always interested in hearing about new projects, so if you'd like to chat please get
+            in touch.</p>
+
+          <!-- form section -->
+
+          <form class="contact-form">
+            <ul class="form-section">
+              <li>
+
+                <input type="text" for="name" name="name">
```

Comment:
- [ ] Your input name should accept up to 30 chars as instructed in the project requirements. Validating your input is essential in enhancing the user experience and improving the reliability of the data provided by users 📜📜📜
To do that you can set the attribute <code>maxlength</code> to **30**